RACK-Ontology/ontology/PROV-S.sadl

```diff
@@ -10,41 +10,57 @@
 	dataInsertedBy (note "The activity that caused this data to be added to RACK") describes THING with values of type ACTIVITY.
+	dataInsertedBy describes THING with at most one value of type ACTIVITY.
 
 ENTITY (note "An entity is a physical, digital, conceptual, or other kind of thing with some fixed aspects; entities may be real or imaginary.")
 	is a type of THING.
 
 	wasDerivedFrom (note "A derivation is a transformation of an entity into another, an update of an entity resulting in a new one, or the construction of a new entity based on a pre-existing entity.") describes ENTITY with values of type ENTITY.
-	wasRevisionOf (note "Indicates a direct line of primary descendancy from one entity to a derivative entity.") describes ENTITY with values of type ENTITY.
-	wasRevisionOf is a type of wasDerivedFrom.
-	wasImpactedBy (note "Indicates that an entity materially affected another entity, and changing the former might affect or invalidate the latter.") describes ENTITY with values of type ENTITY.
-	wasImpactedBy is a type of wasDerivedFrom.
-
+	wasRevisionOf (note "Indicates a direct line of primary descendancy from one entity to a derivative entity.") is a type of wasDerivedFrom.
+	wasImpactedBy (note "Indicates that an entity materially affected another entity, and changing the former might affect or invalidate the latter.") is a type of wasDerivedFrom.
 	wasGeneratedBy (note "Generation is the completion of production of a new entity by an activity. This entity did not exist before generation and becomes available for usage after this generation.")describes ENTITY with values of type ACTIVITY.
+	wasGeneratedBy describes ENTITY with at most 1 value.
 	wasAttributedTo (note "Attribution is the ascribing of an entity to an agent. https://www.w3.org/TR/prov-dm/#concept-attribution" ) describes ENTITY with values of type AGENT.
 	generatedAtTime (note "The time this entity was created and available for use") describes ENTITY with values of type dateTime.
+	generatedAtTime describes ENTITY with at most 1 value.
 	invalidatedAtTime (note "The time at which this entity was no longer usable") describes ENTITY with values of type dateTime.
+	invalidatedAtTime describes ENTITY with at most 1 value.
 
 COLLECTION (note "A collection is an entity that provides a structure to some constituents, which are themselves entities. These constituents are said to be member of the collections.")
 	is a type of ENTITY.
 	content (note "A collection is an entity that provides a structure to some constituents, which are themselves entities. These constituents are said to be member of the collections.") describes COLLECTION with values of type ENTITY.
 
 AGENT (note "An agent is something that bears some form of responsibility for an activity taking place, for the existence of an entity, or for another agent's activity.")
 	is a type of THING.
+	agentName describes AGENT with values of type string.
+	agentName describes AGENT with at most 1 value.
```

Comment:
for agentName should we consider having multiple agentNames for situations like if they wanted to capture first/last name or aliases for things like the same person had a name change (i.e. maiden names)

Reply:
Proposing to drop agentName altogether in favor of using Title in the base class.